docs/moderator-handbook.md

```diff
@@ -210,33 +210,28 @@ Here's how moderators deal with violations of our [Code of Conduct](https://code
 1. **Make sure it was intended to violate the Code of Conduct.**
    Not all violations of the CoC were intended as such. A new camper might post a large amount of code for help, unaware that this can be considered spamming. In these cases, you can just ask them to paste their code with services like Codepen or Pastebin.
 
-2. **If the camper clearly violates the Code of Conduct, the moderator will proceed as follows:**
+2. **If the camper clearly and intentionally violates the Code of Conduct, the moderator will proceed as follows:**
```

Comment:
How do you tell if someone has done something intentionally?

Reply:
Technically, you can't - at least not without someone saying "I'm breaking the rules".

Practically, it tends to be handled situationally. A user who shares a video they made about React hooks in response to someone asking about how to start implementing hooks is typically given a reminder that this falls under self-promotion, where a user who drops a random link to their facebook in multiple channels is met with more heavy-handed moderation.

Reply:
You can usually infer intentionality from how people respond to you pointing out the code of conduct violation. A person who didn't do so intentionally will usually be surprised and apologetic, whereas a someone who did so intentionally will be casual or dismissive.